lib/assignDisabledRanges.js

```diff
@@ -20,7 +20,7 @@ const ALL_RULES = "all"
 module.exports = function (
   root/*: Object*/,
   result/*: Object*/
-) {
+)/*: Object*/ {
```

Comment:
Does this always return an object? If so, what object? Is there a more precise annotation we can give?